aspnetcore/mvc/models/file-uploads/samples/5x/largeFiles/LargeFilesSample/Controllers/FileUploadController.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.WebUtilities;
+using Microsoft.Extensions.Logging;
+using Microsoft.Net.Http.Headers;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace LargeFilesSample.Controllers
+{
+    /// <summary>
+    /// controller for upload large file
+    /// </summary>
+    [ApiController]
+    [Route("[controller]")]
+    public class FileUploadController : ControllerBase
+    {
+        private readonly ILogger<FileUploadController> _logger;
+
+        public FileUploadController(ILogger<FileUploadController> logger)
+        {
+            _logger = logger;
+        }
+
+        /// <summary>
+        /// Friendly return model, the position and way of its definition may change in actual use 
+        /// </summary>
+        public class JsonResponse
+        {
+            /// <summary>
+            /// just some message
+            /// </summary>
+            public string Message { get; set; }
+        }
+
+        /// <summary>
+        /// Action for upload large file
+        /// </summary>
+        /// <remarks>
+        /// Request to this action will not trigger any model binding or model validation,
+        /// because this is a no-argument action
+        /// </remarks>
+        /// <returns></returns>
```

Comment:
Could you remove this?

Reply:
If there are any parameters here, it will trigger model binding, I want to clarify this matter